### PR TITLE
Create consent form & Toggle password visibility & Fast Call

### PR DESCRIPTION
**Pull request by PormpimonPang**

### CODE REVIEW

File: lib/pages/chat_page.dart [hunk 1]
```diff
@@ -226,4 +244,92 @@ class _ChatPageState extends State<ChatPage> {
...
+      },
+      {
+        "MedicalTeamName": "อ.พรนภา นาคโนนหัน",
+        "PhoneNumber": "089-493-3902",
+      },
+      {
+        "MedicalTeamName": "ผศ.ดร. รสสุคนธ์ วาริทสกุล",
+        "PhoneNumber": "081-642-3478",
+      },
+    ];
```
Comment (Thaweesak-sai) [inline lib/pages/chat_page.dart:266]: Can we move this to a new flie as a constant.

File: lib/pages/consent_page.dart [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  @override
+  State<StatefulWidget> createState() => _ConsentPageState();
+}
+
+class _ConsentPageState extends State<ConsentPage> {
+  Consent _result = Consent.defaultValue;
+  String consentText =
+      """ข้าพเจ้ายินดีเจ้าร่วมโครงการวิจัยโดยสมัครใจภายหลังจาหที่ข้าพเจ้าได้รับคำชี้แจงเอกสารข้อมูลสำหรับผู้เข้าร่วมโครงการวิจัย พร้อมด้วยการอธิบายจากผู้วัจัยถึงวัตถุประสงค์ของการวิจัย ระยะเวลาของการทำวิจัย วิธีการวิจัย อันตราย หรืออาการที่อาจจะเกิดขึ้นจากการวิจัยอย่างละเอียด โดยจะได้รับข้อมูลเป็นแนวปฏิบัติเพื่อการฟื้นสภาพหลังผ่าตัดตั้งแต่การเตรียมความพร้อมก่อนการผ่าตัดจนถึงหลังผ่าตัด""";
+  String consentText2 =
+      """ทั้งนี้ข้าพเจ้ามีสิทธิที่จะบอกเลิกเข้าร่วมในโครงการวิจัยเมื่อใดก็ได้ โดยไม่จำเป็นต้องแจ้งเหตุผล และการบอกเลิกการเข้าร่วมวิจับนี้ จะไม่มีผลต่อการรักษาโรคหรือสิทธิอื่นๆ ที่ข้าพเจ้าจะพึงได้รับต่อไป นอกจากนี้ผู้วิจัยรับรองว่าจะเก็บข้อมูลส่วนตัวของข้าพเจ้าเป็นความลับ และจะเปิดเผยได้เฉพาะเมื่อได้รับการยินยอมจากข้าพเจ้าเท่านั้น""";
```
Comment (Thaweesak-sai) [inline lib/pages/consent_page.dart:17]: typo `วิจัย`

**Review by Thaweesak-sai — CHANGES_REQUESTED** [review]

File: lib/pages/consent_page.dart (hunk 2 above)
Comment (PormpimonPang) [reply lib/pages/consent_page.dart:17]: 576b1cf Fixed ka

File: lib/pages/chat_page.dart (hunk 1 above)
Comment (PormpimonPang) [reply lib/pages/chat_page.dart]: 28f70eb Fixed ka

**Review by Thaweesak-sai — APPROVED** [review]
LGTM

**Pull request merged**